src/aiai_eval/config.py

```diff
@@ -71,6 +71,8 @@ class TaskConfig:
             to None.
         supertask (str):
             The supertask of the task, describing the overall type of task.
+        modality str:
+            The modality of the input data.
```

Comment:
Maybe mention what the choices are?

Reply:
Consider making the modalities into an enum, just like `Framework`, to make it clearer.